e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -724,6 +724,61 @@ def test_set_workspace_namespace_bindings(
     output = detproc.check_output(sess, set_binding_cmd + ["--namespace", namespace])
     assert bound_to_namespace in output
 
+    # MultiRM: Valid cluster name, no namespace name, auto-create namespace & resource_quota.
+    if is_multirm_cluster:
+        w_name = uuid.uuid4().hex[:8]
+        output = detproc.check_output(
+            sess,
+            [
+                "det",
+                "w",
+                "create",
+                w_name,
+                "--cluster-name",
+                conf.DEFAULT_RM_CLUSTER_NAME,
+                "--auto-create-namespace",
+                "--resource-quota",
+                "cpu=1,memory=1G",
+            ],
+        )
+        assert bound_to_namespace in output
+    # SingleRM: No cluster name, no namespace name, auto-create namespace & resource quota.
+    else:
+        w_name = uuid.uuid4().hex[:8]
+        output = detproc.check_output(
+            sess,
+            [
+                "det",
+                "w",
+                "create",
+                w_name,
+                "--auto-create-namespace",
+                "--resource-quota",
+                "cpu=1,memory=1G",
```

Comment:
The resource quota is supposed to be an integer eg `--resource-quota 5`. It can't take CPU or memory, the integer is inherently used as the GPU request limit!

Reply:
Fair point!

Reply:
Awesomee thanks!